app/assets/stylesheets/mobile.css.scss

```diff
@@ -236,6 +236,11 @@ table.c {
 
 input#todo_description, input#tag_list, textarea#todo_notes, select#todo_project_id, select#todo_context_id {
     width: 95%;
+    padding: 8px 8px;
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Shorthand form for property `padding` should be written more concisely as `8px` instead of `8px 8px`<br>Each selector in a comma sequence should be on its own line